assets/aws/Makefile

```diff
@@ -136,4 +136,4 @@ create-update-pr: update-ami-ids-terraform
 	git checkout -b $(AUTO_BRANCH_NAME)
 	git commit -am "[auto] Update AMI IDs for $(TELEPORT_VERSION)"
 	git push --set-upstream origin $(AUTO_BRANCH_NAME)
-	gh pr create --fill --label=automated --label=terraform --label=no-changelog
+	gh pr create --fill --label=automated --label=terraform --label=no-changelog --reviewer=$(AMI_PR_REVIEWER)
```

Comment:
This needs to be expanded so that `--reviewer` is optional only if `$AMI_PR_REVIEWER` is supplied.

Reply:
`$(if $(AMI_PR_REVIEWER),--reviewer=$(AMI_PR_REVIEWER))` is the most straightforward way. If the var is empty, nothing will get put on the command line

Reply:
🤯

Reply:
I think you can do something like this, replacing line 139:
```
ifeq ($(AMI_PR_REVIEWER),)
gh pr create --fill --label=automated --label=terraform --label=no-changelog
else
gh pr create --fill --label=automated --label=terraform --label=no-changelog --reviewer=$(AMI_PR_REVIEWER)
endif
```

There is probably a more idiomatic way (like appending the arg to a "arg" variable), but the above is probably sufficient.